includes/class-wp-document-revisions-admin.php

```diff
@@ -212,14 +216,13 @@ public function add_help_tab() {
 
 
 	/**
-	 * Helper function to provide help text as either an array or as a string.
+	 * Helper function to provide help text as an array.
 	 *
 	 * @since 1.1
-	 * @param object $screen (optional) the current screen.
-	 * @param bool   $return_array (optional) whether to return as an array or string.
-	 * @returns array|string the help text
+	 * @param WP_Screen $screen (optional) the current screen.
+	 * @returns array the help text
 	 */
-	public function get_help_text( $screen = null, $return_array = true ) {
```

Comment:
Since this is a public function, is there any chance that this is going to be a breaking change?

Reply:
Firstly I changed it because it reduces the amount of code - and automatically increases the coverage!
Of course, the real question is the purpose of the return_array parameter.
The help text mechanism changed in WP3.3 with the text prior being a string and from that version to be an array.
So the actual answer is that there will be a problem here IF you load the plugin on version 3.2 or earlier.
3.3 was released 10 years ago - and I believe that there will be other more significant issues which is why I removed it.
Finally, had the routine been called with return_array set to false, it would not have worked from V3.0.1 onwards!